bootstrap/Makefile

```diff
@@ -61,6 +61,7 @@ docker/run: docker/build
 	-e AWS_DEFAULT_REGION=$(AWS_DEFAULT_REGION) \
 	-e AWS_PROFILE=$(AWS_PROFILE) \
 	-e AWS_SHARED_CREDENTIALS_FILE=/.aws/credentials \
+	-e INCLUDE=/ci-harness/$(PROJECT_NAME)/Makefile \
```

Comment:
To be sure I understand, this is actually including the calling project's Makefile _within_ the context of the running container? So, something like this works:

```
make docker/run target=foo
```

where `foo` is defined in the project's Makefile, not in tardigrade-ci? And for some reason we want to do this, instead of just running `make foo`?

Reply:
@userhas404d ...